resources/js/components/DetailField.vue

```diff
@@ -1,21 +1,11 @@
 <template>
-    <PanelItem :field="field">
-        <template #value>
-            <a :href="`tel:+${unmaskedValue}`" v-if="field.value" class="no-underline text-primary">
-                {{ field.value }}
-            </a>
-            <p v-else>&mdash;</p>
-        </template>
+    <PanelItem :index="index" :field="field">
```

Comment:
I think we need to copy the unmasked value (without spaces), so you can copy the nova components `js/fields/Detail/EmailField.vue`